src/lib/key_store.c

```diff
@@ -142,26 +103,16 @@ rnp_key_store_load_keys(rnp_t *rnp, char *homedir)
 
     if (((rnp_key_store_t *) rnp->pubring)->keyc < 1) {
         fprintf(io->errs, "pub keyring is empty\n");
-        return 0;
-    }
-
-    /* If a userid has been given, we'll use it. */
-    if ((userid = rnp_getvar(rnp, "userid")) == NULL) {
-        /* also search in config file for default id */
-        memset(id, 0, sizeof(id));
-        conffile(rnp, homedir, id, sizeof(id));
```

Comment:
I moved this logic to the CLI level, it is more suitable place then lib level. Please see the rnp_cfg_get_defkey.